kernel/mod_mgmt/src/lib.rs

```diff
@@ -45,6 +45,7 @@ pub use crate_metadata::*;
 
 pub mod parse_nano_core;
 pub mod replace_nano_core_crates;
+pub mod serde;
```

Comment:
do we want this to be public? I don't think it should be accessible by other crates.

Reply:
`serialize_nano_core` relies on the types from `serde`. I'll still make the loading functions `pub(crate)`.